src/views/Status.vue

```diff
@@ -33,6 +33,9 @@
           <div class="mt-4 mb-0 status text-center"
                :class="activeMessage.activeMessageStyle">
             {{ activeMessage.statusMessage }}
+            <v-row v-if="showTimeLeft" class="mt-1 mb-0 text-center d-flex justify-center">
+              <p class="subtitle blueish">Estimated time: {{releaseTimeText}}</p>
```

Comment:
Maybe we should add (besides of the estimated time's calculation) the remaining blocks to finish the pegout.
I think this will be very helpful information to users since is not an approximation, but it is accurate and continuously updated.